scripts/build_industrial_production_per_country_tomorrow.py

```diff
@@ -39,11 +39,14 @@
 
     al_primary_fraction = get(config["Al_primary_fraction"], investment_year)
     fraction_persistent_primary = al_primary_fraction * total_aluminium.sum() / production[key_pri].sum()
-    
+
     production[key_pri] = fraction_persistent_primary * production[key_pri]
     production[key_sec] = total_aluminium - production[key_pri]
 
-    production["Basic chemicals (without ammonia)"] *= config['HVC_primary_fraction']
+    production.insert(4, "HVC (mechanical recycling)", config["HVC_mechanical_recycling_fraction"]*production["HVC"])
+    production.insert(5, "HVC (chemical recycling)", config["HVC_chemical_recycling_fraction"]*production["HVC"])
```

Comment:
```suggestion
    production["HVC (mechanical recycling)"] = config["HVC_mechanical_recycling_fraction"] * production["HVC"])
    production["HVC (chemical recycling)"] = config["HVC_chemical_recycling_fraction"] * production["HVC"])
```
Question is if we want to manually maintain the ordering.